x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -168,7 +167,7 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
     });
   });
 
-  it('Page filters are loaded with custom values provided in the URL', () => {
+  it('should load page filters with custom values provided in the URL', () => {
```

Comment:
we're using a `context` for the 4 tests above and some tests below, would it make sense to wrap this one and the next (line 191) under a `context` related to url testing?

Reply:
if you decide to wrap these 2 tests, you could actually argue that they could live in another `describe` as we're visiting a new url and waiting for alerts... I can see this being a scenario where we have a second `describe` at the root of the file that would have its own `beforeEach`? Thoughts?

Reply:
Actually, I had thought about creating a group of URL tests, but URL tests can also belong to a different, more imporant group such as context `Alert Page Filters Customization` contains a URL based test as well.. That is why i did not create a separate category.

Other than that, I guess in future if we add more tests based on URL, I will happy to create a separate `describe` preferably in a separate file. Because multiple `describe` in single file are hard work for example, when you want focused test.